Formula/f/fsarchiver.rb

```diff
@@ -0,0 +1,61 @@
+class Fsarchiver < Formula
+  desc "File system archiver for linux"
+  homepage "http://www.fsarchiver.org"
+  url "https://github.com/fdupoux/fsarchiver/releases/download/0.8.7/fsarchiver-0.8.7.tar.gz"
+  sha256 "f7cd059e70fc57e888db282c622ec050c0dcdbaacc65e3c1eb163cd9d92d810d"
+  license "GPL-2.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "bzip2"
+  depends_on "e2fsprogs"
+  depends_on "libgcrypt"
+  depends_on "lz4"
+  depends_on "lzo"
+  depends_on "xz"
+  depends_on "zlib"
+  depends_on "zstd"
+
+  on_macos do
+    depends_on "gettext"
+  end
+
+  on_linux do
+    depends_on "util-linux"
+  end
+
+  def install
+    args = [
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+      "--disable-e2initrd-helper",
+      "MKDIR_P=mkdir -p",
+    ]
+    args += if OS.linux?
+      %w[
+        --enable-elf-shlibs
+        --disable-fsck
+        --disable-uuidd
+        --disable-libuuid
+        --disable-libblkid
+        --without-crond-dir
+      ]
+    else
+      ["--enable-bsd-shlibs"]
+    end
+
+    ENV["HOMEBREW_CCCFG"] = "sa"
+    system "autoreconf", "--force", "--install", "--verbose"
+    system "./configure", "--prefix=/usr", *args
+
+    system "/home/linuxbrew/.linuxbrew/bin/make"
+    system "/home/linuxbrew/.linuxbrew/bin/make", "install"
```

Comment:
```suggestion
    system "make"
    system "make", "install"
```
These paths don't work on macOS.